From 066e1d364baef67e9cae2289dbb47d6a0724a360 Mon Sep 17 00:00:00 2001
From: Shourai <10200748+Shourai@users.noreply.github.com>
Date: Sat, 18 Jul 2020 12:50:44 +0200
Subject: [PATCH 5/5] Apply keybindings for zoom

---
 config.def.h | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/config.def.h b/config.def.h
index 1e55784..828ee5f 100644
--- a/config.def.h
+++ b/config.def.h
@@ -210,6 +210,13 @@ static Shortcut shortcuts[] = {
 	{ TERMMOD,              XK_Y,           selpaste,       {.i =  0} },
 	{ ShiftMask,            XK_Insert,      selpaste,       {.i =  0} },
 	{ TERMMOD,              XK_Num_Lock,    numlock,        {.i =  0} },
+	{ MODKEY|ShiftMask,     XK_Up,          zoom,           {.f = +1} },
+	{ MODKEY|ShiftMask,     XK_Down,        zoom,           {.f = -1} },
+	{ MODKEY|ShiftMask,     XK_K,           zoom,           {.f = +1} },
+	{ MODKEY|ShiftMask,     XK_J,           zoom,           {.f = -1} },
+	{ MODKEY|ShiftMask,     XK_plus,        zoom,           {.f = +1} },
+	{ MODKEY|ShiftMask,     XK_underscore,  zoom,           {.f = -1} },
+	{ MODKEY|ShiftMask,     XK_parenright,  zoomreset,      {.f =  0} },
 };
 
 /*
@@ -251,6 +258,9 @@ static uint ignoremod = Mod2Mask|XK_SWITCH_MOD;
  */
 static Key key[] = {
 	/* keysym           mask            string      appkey appcursor */
+    { XK_parenright, Mod1Mask|ShiftMask,"\033[41;6u",    0,    0},
+    { XK_underscore, Mod1Mask|ShiftMask,"\033[95;4u",    0,    0},
+    { XK_plus,       Mod1Mask|ShiftMask,"\033[43;4u",    0,    0},
 	{ XK_KP_Home,       ShiftMask,      "\033[2J",       0,   -1},
 	{ XK_KP_Home,       ShiftMask,      "\033[1;2H",     0,   +1},
 	{ XK_KP_Home,       XK_ANY_MOD,     "\033[H",        0,   -1},
-- 
2.27.0

